lib/lib_phpQuery.php

```diff
@@ -2115,22 +2132,24 @@ protected function pseudoClasses($class) {
 			break;
 			case 'enabled':
 				$this->elements = $this->map(
-					create_function('$node', '
-						return pq($node)->not(":disabled") ? $node : null;')
+					function($node) {
+						return pq($node)->not(":disabled") ? $node : null;
+					}
 				)->elements;
 			break;
 			case 'header':
 				$this->elements = $this->map(
-					create_function('$node',
-						'$isHeader = isset($node->tagName) && in_array($node->tagName, array(
+					function($node) {
+						$isHeader = isset($node->tagName) && in_array($node->tagName, array(
 							"h1", "h2", "h3", "h4", "h5", "h6", "h7"
 						));
 						return $isHeader
 							? $node
-							: null;')
+							: null;
+					}
 				)->elements;
 //				$this->elements = $this->map(
-//					create_function('$node', '$node = pq($node);
+//					function($node) { $node = pq($node);
```

Comment:
This commented section should be removed completely.

Reply:
Done.